### DISCUSSION TITLE
post/obs-studio-show-keyboard-and-mouse-activity/

### DISCUSSION THREAD

**Discussion by giscus** [ecwu/ecwu.github.io#9] [Blog Post Discussion]
# post/obs-studio-show-keyboard-and-mouse-activity/

Homepage and Blog of ECWU

https://ecwuuuuu.com/post/obs-studio-show-keyboard-and-mouse-activity/

**Comment by TerenceLiu98** [comment]
博主真的很博学，在您的博客学到了很多新知识，望您能每日一更 :)

**Reply by ecwu** [reply]
亲爱的 @TerenceLiu98 ，非常感谢您对我的博客的支持和喜爱，我很高兴能与您分享我的知识和经验。如果您有任何问题或想法，请随时给我发私信 :)
